backend/Tests/TreeOfAKind.UnitTests/Validators/AddPersonCommandValidatorTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Reflection;
+using AutoFixture;
+using FluentValidation.Results;
+using TreeOfAKind.Application.Command.Trees.AddPerson;
+using TreeOfAKind.Application.Configuration;
+using TreeOfAKind.Domain.Trees;
+using TreeOfAKind.Domain.Trees.People;
+using Xunit;
+
+namespace TreeOfAKind.UnitTests.Validators
+{
+    public class AddPersonCommandValidatorTests : ValidatorTestsBase<AddPersonCommandValidator>
+    {
+        [Fact]
+        public void Validate_AutoFixtureGeneratedData_ValidationPasses()
+        {
+            var command = Fixture.Create<AddPersonCommand>();
+
+            Assert.True(Validator.Validate(command).IsValid);
+        }
+
+        [Fact]
+        public void Validate_TooLongName_ValidationFails()
+        {
+            var command = Fixture.Create<AddPersonCommand>();
+
+            SetPropertyWithReflection(command, nameof(command.Name),
```

Comment:
What's that

Reply:
Now I know and I don't like it, how about setting property _without_ reflection?